src/containers/pages/FocusInvestigation/map/planCompletion/index.tsx

```diff
@@ -69,7 +69,8 @@ export class PlanCompletion extends React.Component<
     const service = new serviceClass(`${OPENSRP_PLANS}`);
 
     // get the plan as it exists in the OpenSRP Server
-    await service.read(planToconfirm.plan_id).then((planPayload: PlanPayload) => {
+    await service.read(planToconfirm.plan_id).then((fullPayload: PlanPayload[]) => {
+      const planPayload = fullPayload[0];
```

Comment:
@p-netm why do we need to make this change?  Has the OpenSRP response changed, or did this comment just never work before (it did).

Reply:
@moshthepitt Before the `planPayload` was assumed to be a single object but on rechecking, it seems like even for a single plan, the response is actually a jsonArray, 
I have no idea why it worked, if it did work, 
maybe what i can do is add a check to make it more robust such that it will handle both json Objects and json array,

Reply:
@p-netm cool thanks.  I think we should not do that workaround.